packages/x11/driver/xf86-input-libinput/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="xf86-input-libinput"
-PKG_VERSION="1.0.1"
-PKG_SHA256="fddec49c115591918475155bf16aaf23017d7f814cee7823a0c11f867aca245b"
+PKG_VERSION="1.2.0"
```

Comment:
Remember that this package isn't built by default so you likely have to force building it.